trustedshops.php

```diff
@@ -40,21 +40,24 @@ class TrustedShops extends Module
 	private $confirmations = array();
 
 	public static $seal_displayed = false;
-	
+
 	private static $template_version;
 
-	private $available_languages = array('en', 'fr', 'de', 'es', 'it', 'pl');
+	private $available_languages = array('en', 'fr', 'de', 'es', 'it', 'pl', 'nl');
 
 	public function __construct()
 	{
 		$this->name = 'trustedshops';
-		$this->tab = 'payment_security';
-		$this->version = '2.2.3';
+		$this->tab = 'advertising_marketing';
+		$this->version = '2.3.1';
 		$this->author = 'silbersaiten';
 		$this->bootstrap = true;
-		
-		self::$template_version = version_compare(_PS_VERSION_, '1.6', '<') ? '1.5' : '1.6';
 
+		self::$template_version = version_compare(_PS_VERSION_, '1.6', '<') ? '1.5' : '1.6';
+		$this->ps_versions_compliancy = array(
+			'min' => '1.5.0.0',
+			'max' => _PS_VERSION_
+		);
```

Comment:
Hi @M03G, we didn't ask to add the variable, just the min and max compatible versions, and that would be 1.5.x -  1.6.0.14